### ISSUE TITLE
Ability to Change All Backgrounds.

### ISSUE THREAD

**Issue by AqilCont** [ppy/osu#5674]
**Describe the new feature:**
I would like a feature in which we can change all backgrounds game-wide to whatever we want. This can solve the problem of beatmaps with disturbing backgrounds without having to stare at a completely black screen. We can also have a cool background on the home screen.

**Proposal designs of the feature:** 
There will be a whole new set of options in Settings saying "Change Home Screen Background", "Change Beatmap Background", and "Change All Backgrounds". When clicking on any one of these, it will spawn a File Picker which lets you pick whatever you want from any of your hard drives. After you pick and click "OK", it will change corresponding backgrounds to the one selected.

**Comment by iiSaLMaN** [comment]
Would have to be a supporter-only feature since it's doing something a supporter can only do.

**Comment by peppy** [comment]
Not sure it needs to be supporter. It could be a setting similar to the menu background (see https://github.com/ppy/osu/pull/5520)

**Comment by iiSaLMaN** [comment]
@peppy In that case, skin backgrounds must be a non-supporter thing for consistency.

**Comment by peppy** [comment]
No. See the implementation. There's already allowances for that.

**Comment by Siiiuuuuu** [comment]
May be nice to have now bundled in with the new song select

**Comment by frenzibyte** [comment]
This has nothing to do with song select redesign efforts.
